### DISCUSSION TITLE
Can't change Melilisearch project

### DISCUSSION THREAD

**Discussion by xavipedrals** [AntlerVC/firetable#1170] [Support / Q&A] [answered]
Hello everyone,
After adding some extra columns to my Firestore DB I saw that Rowy full text search was not fetching that new data so I deleted the project in Melilisearch and created a new one. I need to provide the new Melilisearch URL and API key to Rowy but when I go to Project->Settings->Services I can't do it. See the screenshot below:

<img width="1426" alt="Screenshot 2023-03-22 at 16 02 05" src="https://user-images.githubusercontent.com/6597415/226946787-a5c55d0b-b987-426e-ba96-df7b134b0dc4.png">

The Melilisearch index does not exist anymore because I deleted it manually. How can I change my credentials?

Thanks in advance

**Comment by BeeBombshell** [comment, marked as answer]
Hey @xavipedrals, our team is working on adding an enabling-disabling option for the Full Text Search and some advanced settings, which would include changing your MeiliSearch project. Appreciate your patience!

**Comment by harinij** [comment]
Hi @xavipedrals This has been fixed in the recent release. You can disable and re-enable the index to change the Meilisearch project being used.
